django-verdant/Dockerfile

```diff
@@ -55,7 +55,7 @@ WORKDIR /app
 # RUN npm install
 
 # Install less
-RUN npm install less -g
+RUN npm install less@^3.13.1 -g
```

Comment:
I don't know npm too well, but would that actually constrain it below 4? shouldn't it be <4? I might well be wrong but just worth flagging

Reply:
Should be all compatible (semvar) versions

https://stackoverflow.com/questions/22343224/whats-the-difference-between-tilde-and-caret-in-package-json

> ^ will update you to all future minor/patch versions